zipkin-server/src/main/java/zipkin2/server/internal/ui/ZipkinUiConfiguration.java

```diff
@@ -175,8 +175,8 @@ static String maybeResource(String basePath, Resource resource) throws IOExcepti
       // html-webpack-plugin seems to strip out quotes from the base tag when compiling so be
       // careful with this matcher.
       return content.replaceAll(
-        "<base href=[^>]+>", "<base href=\"" + baseTagValue + "\">"
-      );
+        "/zipkin/", baseTagValue
+      ).replaceAll( "<base href=[^>]+>", "<base href=\"" + baseTagValue + "\">");
```

Comment:
ps we tend to use IntelliJ for verifying maven tests. if you are making the test manually or otherwise...

```bash
# Build the server and also make its dependencies
$ ./mvnw -T1C -q --batch-mode -DskipTests --also-make -pl zipkin-server clean package
# after that, you can repeat this as often as you like to test that class
$ ./mvnw test -Dtest=ZipkinUiConfigurationTest -pl ./zipkin-server
```

Reply:
pushed a commit that does this so we can get a patch out